android/BOINC/app/src/main/assets/ca-bundle.crt

```diff
@@ -0,0 +1,3410 @@
+##
```

Comment:
This file should be removed completely from this PR

Reply:
I remove this file